config/default.cfg

```diff
@@ -150,7 +150,8 @@ cfg$files2export$start <- c("config/conopt3.opt",
                             "modules/35_transport/edge_esm/input/scenSpecEnIntensity.cs4r",
                             "modules/35_transport/edge_esm/input/initialIncoCosts.cs4r",
                             "modules/35_transport/edge_esm/input/annualMileage.cs4r",
-                            "modules/35_transport/edge_esm/input/timeValueCosts.cs4r")
+                            "modules/35_transport/edge_esm/input/timeValueCosts.cs4r",
```

Comment:
If that is related to a new edgeTransport version, might be good to specify the new version in the `DESCRIPTION` file